.config/dotnet-tools.json

```diff
@@ -0,0 +1,30 @@
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "husky": {
+      "version": "0.7.2",
+      "commands": [
+        "husky"
+      ]
+    },
+    "csharpier": {
+      "version": "0.30.6",
+      "commands": [
+        "dotnet-csharpier"
+      ]
+    },
+    "dotnet-sonarscanner": {
+      "version": "9.1.0",
+      "commands": [
+        "dotnet-sonarscanner"
+      ]
+    },
+    "jetbrains.resharper.globaltools": {
```

Comment:
I had no idea this was a thing, it makes me very happy!
It will clash with the csharpier though - I reckon should favour this and remove csharpier as it will respect .editorconfig